app/assets/stylesheets/modules/landing.scss

```diff
@@ -51,7 +52,15 @@ h2 {
 }
 
 .featured-airlines {
+  h2 {
+    color: $pink;
+    font-weight: bold;
+  }
+
   font-family: Lato;
-  padding: 50px;
+  padding-top: 15px;
```

Comment:
Properties should be ordered font-family, padding-bottom, padding-left, padding-right, padding-top, text-align